include/umf/memspace.h

```diff
@@ -44,6 +44,13 @@ umfMemoryProviderCreateFromMemspace(umf_memspace_handle_t hMemspace,
                                     umf_memspace_policy_handle_t hPolicy,
                                     umf_memory_provider_handle_t *hProvider);
 
+///
+/// \brief Retrieves predefined host memspace.
```

Comment:
What does "host" memspace mean? Is it something similar to the default kind in Memkind? Anyway, I think we need a more descriptive name.

Reply:
It's a memspace backed by numa nodes. Perhaps `UMF_MEMSPACE_NUMA` would be better?

Reply:
I think we should call it the same as in memkind - "defaut". In memkind, all NUMA nodes available for allocation (so also, for example, PMEM nodes set to KMEM_DAX mode) created a "default" namespace. In addition, NUMA nodes that contain the CPU (= DRAM memory) created the "regular" kind.

Reply:
I've changed the memspace name to `default`.